### DISCUSSION TITLE
Please help with if/else

### DISCUSSION THREAD

**Discussion by xKotelek** [Rapptz/discord.py#9132] [General] [answered]
I'm doing a command that checks site id and if it in list send a list of parameters (domain, creator, language). But when i do 2 ifs (if the id is in first of list (first id and 3 parameters) and when it's in the third list (2 id same parameters). When i type it that's look like that:
![obraz_2022-12-11_125055711](https://user-images.githubusercontent.com/98203141/206901928-3645aca5-eb77-4d43-a2a4-faae18164bc4.png) or with third id
![obraz_2022-12-11_125120832](https://user-images.githubusercontent.com/98203141/206901946-a81e9eca-a9e5-4204-bd2a-88feb9f0dfa1.png) or with id that didn't be in list 
![obraz_2022-12-11_125146211](https://user-images.githubusercontent.com/98203141/206901965-bcbf0771-7b39-4c8e-ae43-bb94c7a55871.png) or without params ![image](https://user-images.githubusercontent.com/98203141/206901985-a153368c-7c72-4200-a968-f52fd9b768a8.png) . Is there any way to stop action sequence or something like that, that the next scripts (else if it wasn't in the list) can not be started. I can send my list and command code if you want. 
![image](https://user-images.githubusercontent.com/98203141/206902053-402af972-d371-4374-b208-5c225cbd0dd3.png)
![image](https://user-images.githubusercontent.com/98203141/206902059-3cd35771-fbcd-4215-8bda-8eaa1ee6d874.png)

**Comment by LostLuma** [comment, marked as answer]
Hello, you're probably looking to use an `elif` statement instead of several ifs in a row, as it allows only executing the final `else` block if none of the `if` conditions were met. See the [Python tutorial](https://docs.python.org/3/tutorial/controlflow.html#if-statements) and [Python documentation](https://docs.python.org/3/reference/compound_stmts.html#the-if-statement) for more information.

**Reply by xKotelek** [reply]
Thank You!
